git-cliff/src/args.rs

```diff
@@ -229,6 +229,9 @@ pub struct Opt {
     /// Include only the tags that belong to the current branch.
     #[arg(long, help_heading = Some("FLAGS"))]
     pub use_branch_tags: bool,
+    /// Fail on a commit that is not matched by any commit parser.
+    #[arg(short, long, help_heading = Some("FLAGS"))]
+    pub fail_on_unmatched_commit: bool,
```

Comment:
Can we remove the CLI argument for now? I'm not sure if this option will be used common enough that it will need to be activated by the CLI.

Reply:
Then this could be still overridden by a environment variable I believe.

I prefer to wait a bit before landing config options as CLI arguments.

Reply:
hid the arg and added env